src/stactools/viirs/fragment.py

```diff
@@ -53,6 +56,17 @@ def collection_dict(self) -> Dict[str, Any]:
         collection["links"] = [Link.from_dict(link) for link in collection["links"]]
         return collection
 
+    def _update_assets(self, production_year_doy: str) -> None:
+        def update_fields(source: Dict[str, Any], updates: Dict[str, Any]) -> None:
+            for band, fields in updates.items():
+                for field, value in fields.items():
+                    source[band][field] = value
+
+        asset_updates = self._load("assets-updates.json")
+        for update_year_doy, bands in asset_updates.items():
+            if int(production_year_doy) >= int(update_year_doy):
```

Comment:
We can. I knew that `update_year_doy` was going to be a string (it's a dict key value), so I guess I was keeping the two things to be compared consistent in type. But since they have to be converted to be compared, might as well cast to an `int` upstream. Done.